src/docker/composebuilder/index.go

```diff
@@ -1,26 +1,42 @@
 package composebuilder
 
 import (
+	"fmt"
+	"regexp"
+	"strings"
+
 	"github.com/Originate/exosphere/src/config"
 	"github.com/Originate/exosphere/src/types"
 )
 
+// GetDockerComposeProjectName creates a docker compose project name the same way docker-compose mutates the COMPOSE_PROJECT_NAME env var
+func GetDockerComposeProjectName(appName string) string {
+	reg := regexp.MustCompile("[^a-zA-Z0-9]")
+	replacedStr := reg.ReplaceAllString(appName, "")
+	return strings.ToLower(replacedStr)
+}
+
+// GetTestDockerComposeProjectName creates a docker compose project name for tests
+func GetTestDockerComposeProjectName(appName string) string {
+	return GetDockerComposeProjectName(fmt.Sprintf("%stests", appName))
+}
+
```

Comment:
Moved these two methods over from the old `service.go` where it didn't belong anyways